### DISCUSSION TITLE
Player-specific Homes - Where does playerUUID tag come from?

### DISCUSSION THREAD

**Discussion by ArcticFox037** [John-Paul-R/Essential-Commands#230] [Q&A] [answered]
In the player specific .dat files, where does the data contained in the playerUUID tag come from? Trying to transfer homes from a plugin.

**Comment by John-Paul-R** [comment]
That should be the unique account identifier provided by mojang. You can look that up via a site like https://mcuuid.net/

In the java code, that can be extracted from the `ServerPlayerEntity` or `GameProfile`

**Reply by ArcticFox037** [reply]
Found it's actually a little different than that. It's actually stored weirdly. I'll elaborate when I've moved everything over

**Reply by John-Paul-R** [reply]
Ah, I forgot that it gets stored as the numeric form. One moment, I'll give you something that extracts that.

**Reply by ArcticFox037** [reply]
I'll be honest, I'm almost done doing what I'm doing, but if you want to find that then you can ^^"
At least if something's wrong this would speed up the fixing

**Reply by John-Paul-R** [reply, marked as answer]
This should convert the UUID from text form to the numeric form: https://jdoodle.com/a/6pC5

Just input the uuid in the "command line arguments" field

**Reply by John-Paul-R** [reply]
e.g. input/output:

![image](https://github.com/John-Paul-R/Essential-Commands/assets/29154459/909975ad-3650-4b8a-bf28-06c458205bf4)

**Reply by ArcticFox037** [reply]
Thanks a bunch
